src/helperFunctions/formatProjectDataHelper.js

```diff
@@ -12,6 +12,14 @@ export const formatProjectDataHelper = (project) => {
   ]
     .sort((a, b) => a.localeCompare(b))
     .join(', ')
-  const siteCount = [...new Set(project.records.map((record) => record.site_name))].length
-  return { projectName, formattedYears, countries, organizations, siteCount }
+  const surveyCount = project.records.length
```

Comment:
Consider destructuring `project` as it contains properties (i.e. `records`) which are frequently accessed .